extensions/wikia/Forum/ForumController.class.php

```diff
@@ -31,20 +31,30 @@ public function board() {
 
 		parent::index( self::BOARD_PER_PAGE );
 
-		$this->addAssets();
-		$this->response->setVal( 'description', '' );
-		/** @var Wall $wall */
-		$wall = $this->response->getVal( 'wall' );
+		JSMessages::enqueuePackage( 'Wall', JSMessages::EXTERNAL );
+		$this->response->addAsset( 'forum_js' );
+		$this->response->addAsset( 'extensions/wikia/Forum/css/ForumBoard.scss' );
+		$this->response->addAsset( 'extensions/wikia/Wall/css/MessageTopic.scss' );
+
+		// VOLDEV-36: separate monobook styling
+		if ( $this->app->checkSkin( 'monobook' ) ) {
+			$this->response->addAsset( 'extensions/wikia/Forum/css/monobook/ForumMonobook.scss' );
+			$this->response->addAsset( 'extensions/wikia/Forum/css/monobook/ForumBoardMonobook.scss' );
+		}
+
+		$this->addMiniEditorAssets();
+
+		$this->description = '';
 
 		if ( $ns == NS_WIKIA_FORUM_TOPIC_BOARD ) {
 			$board = ForumBoard::getEmpty();
 
-			$this->response->setVal( 'activeThreads', $board->getTotalActiveThreads( $wall->getRelatedPageId() ) );
+			$this->response->setVal( 'activeThreads', $board->getTotalActiveThreads( $this->wall->getRelatedPageId() ) );
 			$this->response->setVal( 'isTopicPage', true );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "isTopicPage" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)